tools/topology/topology2/production/tplg-targets-ace1.cmake

```diff
@@ -29,6 +29,27 @@ HDMI1_ID=6,HDMI2_ID=7,HDMI3_ID=8"
 SDW_AMP_FEEDBACK=false,SDW_SPK_STREAM=Playback-SmartAmp,SDW_DMIC_STREAM=Capture-SmartMic,\
 SDW_JACK_OUT_STREAM=Playback-SimpleJack,SDW_JACK_IN_STREAM=Capture-SimpleJack"
 
+"cavs-sdw\;sof-mtl-rt712-l0\;PLATFORM=mtl,NUM_SDW_AMP_LINKS=1,\
+SDW_AMP_FEEDBACK=false,SDW_SPK_STREAM=Playback-SmartAmp,\
+SDW_JACK_OUT_STREAM=Playback-SimpleJack,SDW_JACK_IN_STREAM=Capture-SimpleJack,\
+HDMI1_ID=3,HDMI2_ID=4,HDMI3_ID=5"
```

Comment:
I don't think this is correct, sof-soundwire is _expected_ to have HDMI PCM deviecs of 5, 6 and 7. UCM will break if this is not true: https://github.com/alsa-project/alsa-ucm-conf/blob/master/ucm2/sof-soundwire/Hdmi.conf

Also to note that the support for HDMI passthrough also relies on this rule: https://github.com/alsa-project/alsa-ucm-conf/pull/411/commits/0397a388a234f259ae043a35f1f5fc700d3d3b56

Reply:
@bardliao, I se, but the user reports that no HDMI audio and speaker is not working, so we are still missing something.